components/style/properties/build.py

```diff
@@ -23,10 +23,12 @@ def main():
         abort(usage)
     product = sys.argv[1]
     output = sys.argv[2]
+    testing = len(sys.argv) > 3 and sys.argv[3] == "testing"
```

Comment:
We're always passing this argument now, right? can you make this just `sys.argv[3] == "testing"`?